src/main/diagnostics/diagnostics.ts

```diff
@@ -87,17 +92,15 @@ export class DiagnosticsManager {
         import('./connectedToInternet'),
         import('./dockerCliSymlinks'),
         import('./rdBinInShell'),
+        import('./kubeContext'),
       ])).map(obj => obj.default);
-      const checkers = (await Promise.all(imports)).flat();
```

Comment:
Not part of this PR, but I would rename `checkers` in `src/main/diagnostics/dockerCliSymlinks.ts` to something more specific, like `checkDockerCLIPlugins`.